### PR TITLE
Added Cyclic Polynomial Hash package

### PR DESCRIPTION
**Pull request by MarcAzar**

### CODE REVIEW

**Comment by narimiran** [comment]
> Sorry new to git :) let me try again

In the mean time, another packages was added to `packages.json`, which is now causing trouble for you. (merge conflicts)

Since you're new to git, here are some suggestions:

1. Don't send pull requests from your master branch, because of conflicts like this. Creating a new branch with your changes will make your life much easier.
2. I see that you tried to resolve merge conflict, but unsuccessfully. Now either you try to fix that, or (maybe easier for you as git beginner) you delete your fork, and then fork the clean version to start from there.

Basically, you need to add
```
  {
    "name": "Cyclic-Polynomial-Hash",
    "url": "https://github.com/MarcAzar/Cyclic-Polynomial-Hash",
    "method": "git",
    "tags": [
      "Cyclic",
      "Hash",
      "BuzHash",
      "Rolling",
      "n-gram"
    ],
    "description": "A Nim implementation of a Cyclic Polynomial Hash, aka BuzHash",
    "license": "MIT",
    "web": "https://github.com/MarcAzar/Cyclic-Polynomial-Hash"
  },
```
to the __latest__ version of `packages.json`.

If you don't succeed in a reasonable time, ping me here and I'll do it for you.
